app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt

```diff
@@ -125,6 +134,14 @@ class AdminPinActivityTest {
     Intents.release()
   }
 
+  @Test
+  fun testAdminPinActivity_hasCorrectActivityLabel(){
+    activityTestRule.launchActivity(createAdminPinActivityIntent())
+    val title =activityTestRule.activity.title
```

Comment:
Incorrect spacing
`val title = activityTestRule.activity.title`

Reply:
@starboi02 Please reply to this comment

Reply:
@rt4914 thanks. I have made this change.